src/errors/dnsRecordsError.ts

```diff
@@ -0,0 +1,46 @@
+import { DnsRecordType } from '../publicTypes';
+type DnsRecordsErrorHandler = (error: DnsRecordsErrorOptions) => string;
+/** Explains DnsRecords Error options */
+type DnsRecordsErrorOptions = {
+  recordType?: DnsRecordType
+};
+
+export enum DnsRecordsErrorCode {
+  InconsistentTtl = "InconsistentTtl",
+  NoTtlFound = "NoTtlFound",
+}
+
+/**
+ * @internal
+ * Internal Mapping object from DnsRecordsErrorCode to a DnsRecordsErrorHandler
+ */
+const HandlersByCode = {
+  [DnsRecordsErrorCode.InconsistentTtl]: (params: DnsRecordsErrorOptions) => `ttl for record ${params.recordType} is different for other records of the same type`,
+  [DnsRecordsErrorCode.NoTtlFound]: (params) => `Ttl record was not present`
```

Comment:
Based on my research 1 day is considered a standard default TTL. I think we can use it as default: https://serverfault.com/questions/7478/recommended-dns-ttl